assets/templates/forgot-password.html

```diff
@@ -71,15 +71,36 @@ <h5>¿NO RECUERDAS TU CONTRASEÑA?</h5>
           <fieldset>
             <ul class="field-group">
               <legend>forgot password</legend>
-              <li class="field-item error">
+              <li class="field-item dp-validate error">
                 <label for="email">Email:</label>
                 <input type="email" name="email" id="email" placeholder="¡Psst! Es tu Email.">
-                <div class="wrapper-errors">
+                <div class="dp-message dp-error-form">
                   <p>¡Ouch! Este campo está vacío</p>
+                  <a href="">¡Ouch! Este campo está vacío</a>
+                  <span>¡Ouch! Este campo está vacío</span>
+                  <button>¡Ouch! Este campo está vacío</button>
                 </div>
+
               </li>
             </ul>
           </fieldset><!-- user and password -->
+          <div class="form-message dp-error">
+            <div class="wrapper-errors">
+              <p>¡Ouch! Mensaje inesperado</p>
+              <!-- <a href="">¡Ouch! Este campo está vacío</a>
```

Comment:
Are this commented messages ok? I asking out of curiosity.

Reply:
the messages can bring any of these elements, they are only commented so that the message is shorter. It's to prove it, I can erase it.